frontend/src/test/test-helpers.js

```diff
@@ -1,21 +1,25 @@
 import { configure, mount } from 'enzyme';
+import { act } from 'react-dom/test-utils';
 import Adapter from 'enzyme-adapter-react-16';
 
 configure({ adapter: new Adapter() });
 
 let wrapper;
-export const mountToDoc = (reactElm) => {
+export const mountToDoc = async (reactElm) => {
   if (!document) {
     // Set up a basic DOM
-    global.document = jsdom('<!doctype html><html><body></body></html>');
+    global.document = jsdom('<!doctype html><html dir="ltr"><body></body></html>');
   }
   if (!wrapper) {
     wrapper = document.createElement('div');
     wrapper.id = 'app';
     document.body.appendChild(wrapper);
   }
-  const container = mount(reactElm);
-  // container.update();
+  let container;
+  await act(async () => {
+    container = mount(reactElm);
+  });
+  container.setProps();
```

Comment:
I'm assuming that's [this](https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/setProps.html). I'm not sure I understand the significance of calling this without arguments. Is it because you want to trigger `componentWillReceiveProps` per the enyzme docs?

Reply:
(Good point -- I don't really know how containers act on `mount()` vs. `componentWillReceiveProps`. I'm not very familiar with `enzyme` or its standard component testing practices, I tend to use "component integration testing" by rendering an entire view, mocking state hydration, letting the UI state settle, and testing the final shebang.)

Reply:
`container.setProps()` might just be cruft here. Good eye!